.travis.yml

```diff
@@ -10,7 +10,6 @@ addons:
 
 python:
   - "3.6"
-  - "3.7"
```

Comment:
You could have added tests for 3.7, I mean, fix why 3.7 is not working not commenting it out